users/service.go

```diff
@@ -577,7 +583,7 @@ func (svc service) OAuthCallback(ctx context.Context, state mgoauth2.State, clie
 			if errors.Contains(err, repoerr.ErrConflict) {
 				return &magistrala.Token{}, errors.New("user already exists")
 			}
-			return &magistrala.Token{}, err
+			return &magistrala.Token{}, errors.Wrap(svcerr.ErrCreateEntity, err)
```

Comment:
This wrap can be remove since `svc.RegisterClient` is present in same user service layer.